source/plugin/api-versions.rst

```diff
@@ -0,0 +1,81 @@
+============
+API-Versions
+============
+
+This page explains which API versions exists and to which Minecraft version their implementations belong to.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *Planned*   | ---          | --             | * *SpongeForge (1.13.x)*                  |
```

Comment:
We could remove the *Planned* and put a release date of TBA instead. I'm not sure how much use it is here.

Reply:
I mainly added this entry so that, if anybody ever searches the docs (or the IRC^^) for 1.13 they will find a reference to something they can then use for further research (aka we didn't miss the MC 1.13 release and we are working on it (soon)).